src/Shared/LayoutRenderers/AspNetLayoutMultiValueRendererBase.cs

```diff
@@ -251,10 +268,51 @@ protected string GetRenderedValueSeparator(LogEventInfo logEvent)
         }
 
         /// <summary>
-        /// Append the value quoted, escape quotes when needed
+        /// Append the quoted name and value separated by a colon
         /// </summary>
         /// <param name="builder"></param>
+        /// <param name="name"></param>
         /// <param name="value"></param>
+        protected static StringBuilder AppendJsonProperty(StringBuilder builder, string name, string value)
```

Comment:
Could `AppendJsonProperty` + `AppendFlatProperty` become private-methods in `AspNetResponseCookieLayoutRenderer.cs` until a second use-case appear?

Reply:
Changed, had to make AppendQuoted protected instead of private for access